themes-default/slim/static/js/core.js

```diff
@@ -33,27 +33,12 @@ const UTIL = {
         }
 
         const body = document.body;
-        $('[asset]').each(function() {
```

Comment:
Be wary. I tried to do that and it broke all kinds of stuff related to assets when combined with `Vue`, `VueInViewport`, `isotope`, `tablesorter`, etc...

Reply:
I can't find `asset=` anywhere in the themes directory so I don't think we have anything left that uses it even in js. 🙏 I could be wrong though.

Reply:
I agree that `asset=` is no longer being used.